src/olympia/lib/settings_base.py

```diff
@@ -31,6 +31,7 @@
     '.mozilla.com',
     '.mozilla.net',
     '.mozaws.net',
+    'nginx'
```

Comment:
this shouldn't be nessecary

Reply:
Without it, it errors out to a 400 response - `DisallowedHost at /api/v4/addons/search/, Invalid HTTP_HOST header: 'nginx'. You may need to add 'nginx' to ALLOWED_HOSTS.`

Reply:
Ah, `ALLOWED_HOSTS = ALLOWED_HOSTS + [SERVICES_DOMAIN, EXTERNAL_SITE_URL]` is incorrect then.  It's because we're adding the full _url_ rather than just the domain. Add `'nginx'` to the list rather than `EXTERNAL_SITE_URL`